client/package.json

```diff
@@ -14,10 +14,11 @@
     "classnames": "^2.3.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
+    "react-scripts": "4.0.3",
```

Comment:
changed to 4.0.3 to fix https://stackoverflow.com/questions/64557638/how-to-polyfill-node-core-modules-in-webpack-5

Reply:
This is pretty bad.